src/draw/vg_lite/lv_draw_vg_lite_label.c

```diff
@@ -243,6 +243,12 @@ static void draw_letter_outline(lv_draw_vg_lite_unit_t * u, const lv_draw_glyph_
     path_clip_area.x2++;
     path_clip_area.y2++;
 
+    /* draw bounding box for debug */
```

Comment:
This code remains and can be used when anyone wants to debug fonts.

Reply:
This is unnecessary. If the feature is used for debugging, you can submit a separate PR.
Let us focus on fixing the freetype bug in this PR.